packages/react-icons/bin/index.js

```diff
@@ -87,5 +88,5 @@ rmSync(outDirPath, {recursive: true, force: true});
 rmSync('./dist', {recursive: true, force: true});
 fs.ensureDirSync(outDirPath);
 
-const svgs = globSync(`${iconsSourceDirPath}/**/*.svg`);
+const svgs = globSync(`${iconsSourceDirPath}/**/*.svg`).map((svgPath) => svgPath.split(path.sep).join(path.posix.sep));
```

Comment:
Why this change?

Reply:
`globSync` returns paths formatted with `\` instead of `/` when running the script on a windows machine, which breaks other parts of the script that the path strings to have `/`. This formats all the paths in the UNIX format.